### ISSUE TITLE
Fonts too small on Windows Server 22 / Windows 11 with HiDPI display in special cases

### ISSUE THREAD

**Issue by mgmax** [t-oster/VisiCut#668]
In certain (unusual?) situations, some text in VisiCut is waaay too small on Windows Server 2022. Probably the same for Windows 11. This is probably a Java bug which we can't do much about. I'm still documenting it here nevertheless.

Possible solutions:
- Update Java version
- try workaround https://stackoverflow.com/questions/71788510/java-font-size-changes-from-win-10-to-win-11

In real life, this occurs in remote-desktop sessions. I have only partially managed to reproduce it in a more standard environment.

- Install Windows Server 2022 Evaluation image in VirtualBox (probably same for Windows 11) https://www.microsoft.com/en-us/evalcenter/download-windows-server-2022
- Install all updates (probably makes no difference)
- Install VisiCut
- Increase display resolution to at least 1280x1024

a) Set display scaling to 150% using "custom scaling" factor
--> VisiCut scales correctly
![grafik](https://user-images.githubusercontent.com/1520891/232547642-7ba9e343-0756-45f5-a3e4-e30bcf26cec5.png)

b) Turn off custom scaling, then set the display scale factor to 100% (same value as before, but different method), then reboot, then set the factor to 150% NOT using custom scaling
--> most text is waaay too small, even smaller than on 100%
![grafik](https://user-images.githubusercontent.com/1520891/232551017-184b6c73-1e4f-4c8c-a33a-bcf53f727d0f.png)

c) Starting from b, reboot (or log out and back in again)
--> everything is fine
![grafik](https://user-images.githubusercontent.com/1520891/232549909-d63c6d58-14b9-4b1b-8dd7-8a6064af2d1e.png)

So basically I could only clearly reproduce this yet if the DPI is changed during the session before VisiCut is started.
